source/usage/rancheros.rst

```diff
@@ -1,3 +1,57 @@
 =========
 RancherOS
 =========
+
+.. contents::
+   :local:
+
+* https://rancher.com/rancher-os/
+
+RancherOS, a lightweight, secure Linux distribution, built from containers to run containers well.
+
+Preparations
+============
+Create a ``clouds.yaml`` and ``secure.yaml`` file (see :ref:`Authentication` chapter).
+
+
+Cloud-config
+============
+Cloud-config is a declarative configuration file format supported by many Linux distributions and is the primary configuration mechanism for ``RancherOS``. A Linux OS supporting cloud-config will invoke a cloud-init process during startup to parse the cloud-config file and configure the operating system. ``RancherOS`` runs its own cloud-init process in a system container. The cloud-init process will attempt to retrieve a cloud-config file from a variety of data sources. Once cloud-init obtains a cloud-config file, it configures the Linux OS according to the content of the cloud-config file.
+
+* https://rancher.com/docs/os/v1.x/en/installation/configuration/#cloud-config
+
+* https://cloudinit.readthedocs.io/en/latest/
+
+* https://cloudinit.readthedocs.io/en/latest/topics/modules.html
+
+
+Openstack Config drive
+======================
+If we change the settings from the ``RancherOS``, we need to enable the config drive parameter. Pass ``--config-drive`` to the ``openstack server create`` command (https://docs.openstack.org/nova/queens/user/config-drive.html).
```

Comment:
Always link the `latest` or `master` version of OpenStack docs

Reply:
no master or latest from the config-drive found.